melodies_monet/driver.py

```diff
@@ -902,6 +902,12 @@ def plotting(self):
             # get the plot type
             plot_type = grp_dict['type']
 
+            #read-in special settings for multi-boxplot
+            if grp == 'plot_grp6':
```

Comment:
This condition should use `plot_type` to check, since the plot group name can be anything.